ambry-api/src/main/java/com/github/ambry/config/QuotaConfig.java

```diff
@@ -94,7 +94,7 @@ public QuotaConfig(VerifiableProperties verifiableProperties) {
    * Build the default quota enforcer and source pair json.
    * @return Json string.
    */
-  private static JSONObject buildDefaultQuotaEnforcerSourceInfoPairJson() {
+  public static JSONObject buildDefaultQuotaEnforcerSourceInfoPairJson() {
```

Comment:
why making it public, it's not used anywhere outside of this class.

Reply:
fixed.